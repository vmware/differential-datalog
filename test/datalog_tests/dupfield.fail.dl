// two fields with the same name
typedef t = Constr { f: bigint, f: bigint }

//---

// Two symbols with the same name
typedef f = bigint
function f(): bigint
