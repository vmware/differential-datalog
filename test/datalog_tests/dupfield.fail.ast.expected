error: ./test/datalog_tests/dupfield.fail.dl:2:22-2:31: Multiple definitions of argument f at the following locations:
  ./test/datalog_tests/dupfield.fail.dl:2:22-2:31
  ./test/datalog_tests/dupfield.fail.dl:2:33-2:43

Failed to parse input file: "./test/datalog_tests/dupfield.fail.dl" (line 6, column 1):
unexpected end of input
expecting "="
